### DISCUSSION TITLE
test -m testPriceFeedVersionIsAccurate error

### DISCUSSION THREAD

**Discussion by QsAdedipe** [Cyfrin/foundry-full-course-cu#1633] [Q&A]
I am getting this error below everytime I run this code 

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/97874921/40800ed0-c94b-43b2-add9-165059fe1b27)

**Comment by EngrPips** [comment]
Hello @QsAdedipe, `-m` is outdated. Please use `--mt` instead.

**Comment by c0deg33k** [comment]
Use ```--mt ``` as @EngrPips says or use the more verbose format ``` --match-test <TEST_FUNCTON_NAME> ```

**Reply by arpit3210** [reply]
Thanks, Works 👍🏻

**Reply by EngrPips** [reply]
Awesome. I am glad it worked for you.

**Comment by vectordotsats** [comment]
Thanks
